spec/models/employee_spec.rb

```diff
@@ -5,6 +5,11 @@
      subject { create(:employee) }
      it { should validate_presence_of(:slack_username) }
      it { should validate_uniqueness_of(:slack_username) }
+     # These fail - why?
+     #it { should allow_values("test_user_1", "testuser2", "test.user.3").for(:slack_username) }
+     #it { should_not allow_values("TEST USER 1", "test_user_2?", "test'user'3").for(:slack_username) }
```

Comment:
In thinking about this a bit more, while this would be an interesting investigation to go down I don't think it's necessary at this time for the sake of finishing this PR and moving forward to other things.  If it rears it's head again we can revisit it then.